### PR TITLE
APEX-385 Skyline adjust smart contracts

### PR DESCRIPTION
**Pull request by MiroslavStefanovic**

### CODE REVIEW

File: contracts/Claims.sol [hunk 1]
```diff
@@ -226,6 +237,7 @@ contract Claims is IBridgeStructs, Initializable, OwnableUpgradeable, UUPSUpgrad
                 uint8 _txType = _ctx.transactionType;
                 if (_txType == 0) {
                     chainTokenQuantity[chainId] += _ctx.totalAmount;
+                    chainTokenQuantity[chainId] += _ctx.totalWrappedAmount;
```
Comment (igorcrevar) [inline contracts/Claims.sol:240]: `chainWrappedTokenQuantity` ?

File: contracts/Claims.sol [hunk 2]
```diff
@@ -234,6 +246,7 @@ contract Claims is IBridgeStructs, Initializable, OwnableUpgradeable, UUPSUpgrad
                         confirmedTransactions[chainId][nextNonce].retryCounter++;
                     } else {
                         chainTokenQuantity[chainId] += _ctx.totalAmount;
+                        chainTokenQuantity[chainId] += _ctx.totalWrappedAmount;
```
Comment (igorcrevar) [inline contracts/Claims.sol:249]: `chainWrappedTokenQuantity` ?

File: contracts/Claims.sol [hunk 3]
```diff
@@ -396,12 +428,23 @@ contract Claims is IBridgeStructs, Initializable, OwnableUpgradeable, UUPSUpgrad
...
-        uint256 _tokenAmount
+        uint256 _chainTokenAmount,
+        uint256 _chainWrappedTokenAmount
     ) external onlyAdminContract {
-        _isIncrease ? chainTokenQuantity[_chainId] += _tokenAmount : chainTokenQuantity[_chainId] -= _tokenAmount;
+        chainTokenQuantity[_chainId] = _isIncrease
+            ? chainTokenQuantity[_chainId] + _chainTokenAmount
+            : chainTokenQuantity[_chainId] - _chainTokenAmount;
+
+        chainWrappedTokenQuantity[_chainId] = _isIncrease
```
Comment (igorcrevar) [inline contracts/Claims.sol:445]: I think we need two `_isIncrease` bools or maybe two functions for each type of tokens

File: contracts/Claims.sol (hunk 2 above)
Comment (MiroslavStefanovic) [reply contracts/Claims.sol]: Fixed, added test.

File: contracts/Claims.sol (hunk 3 above)
Comment (MiroslavStefanovic) [reply contracts/Claims.sol:450]: Done.

File: contracts/Claims.sol [hunk 4]
```diff
@@ -268,10 +281,13 @@ contract Claims is IBridgeStructs, Initializable, OwnableUpgradeable, UUPSUpgrad
         if (_quorumReached) {
             uint8 chainId = _claim.chainId;
             uint256 changeAmount = _claim.amount;
+            uint256 changeWrappedAmount = _claim.amountWrapped;
```
Comment (igorcrevar) [inline contracts/Claims.sol:284]: We need for HotWalletIncrement claim two increase bools

**Comment by igorcrevar** [comment]
Add  to admin
```
function getChainWrappedTokenQuantity(uint8 _chainId) external view returns (uint256) {
        return claims.chainWrappedTokenQuantity(_chainId);
    }
```

**Comment by igorcrevar** [comment]
We need to change this function inside claims 
```function _submitClaimHWIC(HotWalletIncrementClaim calldata _claim, address _caller) internal {```
`HotWalletIncrementClaim` should have increment bool for wrapper token also
Both quantities should be included in one claim, but we need separate increments/decrements

**Review by igorcrevar — APPROVED** [review]

**Pull request merged**